src/coreclr/src/dlls/mscoree/unixinterface.cpp

```diff
@@ -133,6 +134,11 @@ static void ConvertConfigPropertiesToUnicode(
             // is passed in as the value of "BUNDLE_PROBE" property (encoded as a string).
             *bundleProbe = (BundleProbe*)_wcstoui64(propertyValuesW[propertyIndex], nullptr, 0);
         }
+
+        if (strcmp(propertyKeys[propertyIndex], "EMBEDDED_RUNTIME") == 0)
```

Comment:
Do we really need to pass this as a key/value pair? Can we just use a static bool variable in one of those files that is compiled differently for the static host?

Reply:
I figured if that was possible, we would use that mechanism for BUNDLE_PROBE as well.

Reply:
Or you mean a file that compiles differently in coreclr? I guess that might work.

Reply:
`BUNDLE_PROBE` is independent on statically linked self-contained host. It works even when `libcoreclr.so` is in separate library.

Reply:
When we discussed this it was about "host passes this as a parameter" so I kind of latched to that solution. 
Yes, since we have files compiled differently between embedded and regular coreclr, I could just put some static that has a different value.

Reply:
Renamed the variable `g_coreclr_embedded` with value known at compile time. Another good point is that it is now entirely in VM and PAL does not need to be involved in determining which mode we run.